Add a readme

dwitterer opened the pull request:
This is a good theme and all,  but without any readme or description, noone will know how to use it. I know this, because I had the same exact problem just a moment ago. So, I wrote a readme for you! :D

dwitterer committed:
Create README.md

dwitterer committed:
Update README.md

dwitterer committed:
Update README.md

dwitterer committed:
Update README.md

dwitterer committed:
Update README.md

duneheishman commented:
Awesome readme, thanks! It definitely needed one. Only two things: the icon paths in the example boot stanzas are wrong, and it looks like the code indentation isn't showing up properly when rendered on Github. If you can fix up those two things I'll merge it

dwitterer committed:
Touched up and updated readme

The author of the parent repo can merge it now...

dwitterer commented:
Hey, sorry, I didn't see your reply till now. I fixed the indentation and updated the boot stanzas. I assume that the user *is* supposed to put the `highres` folder in the stanzas, or are they just supposed to copy/download one of the folders into their themes directory?

Anyways, if you're happy, you can merge this. And if not, then merge it and modify it yourself I guess.

P.S Since I'm here - do you know a good place to go to for rEFInd font files? I seem to be unable to find any resources for rEFInd in general (currently in the hard process of creating my own icons in Inkscape :P), but font files seem to be the hardest to find... I tried using the mkfont script, but that failed miserably :/

Thanks,
Dwitterer
